website/index.html

```diff
@@ -677,6 +677,23 @@ <h2 id="extensions"><a href="#extensions">Extensions</a></h2>
         <p>You can also take a look at
           <a href="https://github.com/bnb/awesome-hyper">Awesome Hyper</a>
           for a curated list of plugins and resources.</p>
+          
+          <h2 id="keymaps"><a href="#keymaps">Keymaps</a></h2>
+    
+          <P> All command keys can be changed. In order to change them, edit 
```

Comment:
Maybe add a link to default keymap files on 'All command keys' to list all available commands?